src/creature_control.h

```diff
@@ -348,7 +348,7 @@ unsigned char sound_flag;
     MapSubtlCoord targtstl_x;
     MapSubtlCoord targtstl_y;
     unsigned long instance_use_turn[INSTANCE_TYPES_MAX];
-    char instance_available[INSTANCE_TYPES_MAX];
+    short instance_available[INSTANCE_TYPES_MAX];
```

Comment:
this simply stores a yes or no INSTANCE_TYPES_MAXs times, the size of the yes or no can remain small, a TbBool makes more sense then a char though